apriltag_ros/include/apriltag_ros/continuous_detector.h

```diff
@@ -69,6 +72,20 @@ class ContinuousDetector: public nodelet::Nodelet
   void refreshTagParameters();
 
  private:
+  image_geometry::PinholeCameraModel camera_model_;
+  tf::Transform tag_pose_;
+  std::string base_frame_id_;
+  tf::TransformListener tf_listener_;
+
+  double fov_size_scaler_{ 1.0 }; // scales the size of the image to produce a larger or smaller FOV size (acts as an FOV 'buffer')
+  // squared version of min and max detection distances (set these limits widely to not conflict with navigation)
+  double min_detection_dist_{ 0.0 }; // Minimum euclidean distance in 3d for the detector to run
+  double max_detection_dist_{ 10.0 }; // Maximum euclidean distance in 3d for the detector to run
```

Comment:
```suggestion
  double max_detection_dist_{ 10.0 }; // Maximum distance in 3d for the detector to run
```

since we're only checking one dimension now, maybe that still technically euclidean, but IMO we can remove the word now

Reply:
yes I agree